src/opm/common/utility/TimeService.cpp

```diff
@@ -146,6 +169,25 @@ std::time_t mkdate(int in_year, int in_month, int in_day) {
     return mkdatetime(in_year , in_month , in_day, 0,0,0);
 }
 
+// The portable_timegm() function is based on
+// https://stackoverflow.com/questions/16647819/timegm-cross-platform
+// answer by Sergey D.
+std::time_t portable_timegm(struct tm const* t)
```

Comment:
Can we write `std::tm` instead of `struct tm` here?  I appreciate that the original source probably was a C project, but we're a C++ project.